src/pages/Home/MainPrecaution/index.tsx

```diff
@@ -1,142 +1,145 @@
 import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Collapse, ListItem, Divider } from '@chakra-ui/react'
+
 import {
-  Heading,
-  Button,
-  Collapse,
-  Box,
-  ListItem,
-  List,
-  Link,
-  Divider,
-  Text,
-} from '@chakra-ui/react'
-import { IoMdArrowDropdownCircle } from 'react-icons/io'
-import { Inner, StyledDiv3, CollapseInner, Precaution } from './styles'
+  StyledDiv3,
+  CollapseInner,
+  Precaution,
+  PrecautionText,
+  PrecautionBtns,
+  PrecautionLeftBtn,
+  PrecautionMessage,
+  PrecautionRightBtn,
+  PrecautionIcon,
+  LeftBox,
+  RightBox,
+  PrecautionLink,
+  Inner,
+  PrecautionList,
+  ButtonInner,
+  MainPetitionBtn,
+} from './styles'
 
 const MainPrecaution = (): JSX.Element => {
   const [leftOpen, setLeftOpen] = useState(false)
   const [rightOpen, setRightOpen] = useState(false)
-  const ButtonStyle = {
-    borderRadius: '0',
-    height: '54px',
-    width: '50%',
-    display: 'inline-flex', // Inner 900px에 글자를 딱 맞추기 위해 flex 값을 주었습니다.
-    justifyContent: 'space-between',
+  const petitionNavigate = useNavigate()
+  const onPetitionBtn = () => {
+    petitionNavigate('/write')
   }
```

Comment:
```suggestion
  const navigate = useNavigate()
  const onPetitionBtn = () => {
    navigate('/write')
  }
```
별도의 추가 기능을 하지 않는다면 navigate 으로 하는게 좋을 것 같아요!